.packer/win/windows-build-server.json

```diff
@@ -0,0 +1,90 @@
+{
+    "variables": {
+        "aws_access_key": "",
+        "aws_secret_key": "",
+        "instance_type": "c5.2xlarge",
+        "windows_password": ""
+    },
+    "builders": [
+        {
+            "force_deregister": true,
+            "force_delete_snapshot": true,
+            "type": "amazon-ebs",
+            "access_key": "{{user `aws_access_key`}}",
+            "secret_key": "{{user `aws_secret_key`}}",
+            "region": "eu-west-1",
+            "instance_type": "{{user `instance_type`}}",
+            "source_ami_filter": {
+                "filters": {
+                    "name": "Windows_Server-2019-English-Full-Base-*"
+                },
+                "owners": ["801119661308"],
```

Comment:
It would be great to know what does this number mean.

Reply:
801119661308 - it is amazon shop for ami https://thecloudmarket.com/owner/801119661308--amazon , 
base image taken from them.